src/implementation/Client/DaprClient.ts

```diff
@@ -52,15 +53,15 @@ export default class DaprClient {
   readonly actor: IClientActorBuilder;
 
   constructor(
-    daprHost: string
-    , daprPort: string
+    daprHost = Settings.getDefaultHost()
```

Comment:
I am not a fan personally of this line, the below follows a different way of working than this. Could we make it so constructor has all optional parameters then and we add the same `??` to daprHost as welll on line 63?

Reply:
Good suggestion, that will look more uniform!

Reply:
For `GRPCClient` and `HTTPClient`, I have still kept it inline as that looks easier to read. Let me know what your thoughts are, we can change that too.

Reply:
Ok for me